metrique/src/lib.rs

```diff
@@ -403,4 +403,26 @@ pub use metrique_metricsrs as metrics_rs;
 
 pub use metrique_core::concat;
 
-pub use metrique_writer as writer;
+/// Re-exports of [metrique_writer]
+pub mod writer {
+    pub use metrique_writer::GlobalEntrySink;
+    pub use metrique_writer::{AnyEntrySink, BoxEntrySink, EntrySink};
+    pub use metrique_writer::{BoxEntry, EntryConfig, EntryWriter, core::Entry};
+    pub use metrique_writer::{Convert, Unit};
+    pub use metrique_writer::{EntryIoStream, IoStreamError};
+    pub use metrique_writer::{MetricFlags, MetricValue, Observation, Value, ValueWriter};
+    pub use metrique_writer::{ValidationError, ValidationErrorBuilder};
+
+    // Use the variant of the macro that has `metrique::` prefixes.
+    pub use metrique_writer_macro::MetriqueEntry as Entry;
+
+    pub use metrique_writer::AttachGlobalEntrySinkExt;
+
+    #[cfg(feature = "test-util")]
+    pub use metrique_writer::test_util;
+    pub use metrique_writer::{entry, format, sample, sink, stream, value};
+
+    #[doc(hidden)]
+    pub use metrique_writer::core;
+    pub use metrique_writer::{AttachGlobalEntrySink, EntryIoStreamExt, FormatExt, unit};
```

Comment:
`unit` should be `doc(hidden) in this position since its already ex-exported from metrique.